test/bigtest/interactors/package-search.js

```diff
@@ -33,7 +33,11 @@ import SearchBadge from './search-badge';
   hasErrors = isPresent('[data-test-query-list-error="packages"]');
   errorMessage = text('[data-test-query-list-error="packages"]');
   noResultsMessage = text('[data-test-query-list-not-found="packages"]');
-  selectedSearchType = collection('[data-test-search-form-type-switcher] a[class^="is-active--"]');
+  selectedSearchType = collection([
+    '[data-test-search-form-type-switcher] a[class^="primary--"]',
+    '[data-test-search-form-type-switcher] a[class*=" primary--"]'
```

Comment:
Yeah, it's a clunky selector since we shouldn't rely on what order the classes appear in. `primary-{hash}` could be first, or later in the collection of classes on the button.